content/en/apps/guides/data/analytics/setup.md

```diff
@@ -65,4 +65,4 @@ docker compose -f docker-compose.couchdb.yml -f docker-compose.postgres.yml -f d
 To remove all the data volumes, add `-v` at the end of this command.
 
 ## Setup Superset
-To build data visualization dashboards, follow the [Superset instructions](https://superset.apache.org/docs/installation/installing-superset-using-docker-compose/) to run Superset and connect it to the PostgreSQL database.
+To build data visualization dashboards, follow the [Superset instructions](https://superset.apache.org/docs/installation/installing-superset-using-docker-compose/) to run Superset and connect it to the PostgreSQL database. Medic recommends adding a way to track Superset changes via a git repository or any other version control system to make it easier to track changes over time and potentially catch and remediate bugs and regressions. One way to do this is by using [Superset API](https://superset.apache.org/docs/api/) to export and import dashboard and chart definitions. This [guide](https://www.restack.io/docs/superset-knowledge-apache-superset-dashboard-code) provides instructions on how to manage Superset dashboards as code and is a good starting point.
```

Comment:
Should we mention an alternative way of doing this i.e. adding the SQL queries used to create the widgets? It causes some confusion IMO and is not ideal because people have to remember to update the queries. The suggested way also some shortcomings as @dianabarsan mentions in this [slack](https://medic.slack.com/archives/C03SP4KRVEX/p1722624243704319?thread_ts=1722347352.446559&cid=C03SP4KRVEX) comment.

Reply:
There is a separate ticket to define a recommended way to capture Superset changes. For now, I think this is sufficient as a starting point.

Reply:
OMG. I have strong opinions on this.
I think the page you linked is probably the only place I found that mentions dashboard as code, and gives a 3 line example. LOL. 
I don't think this is enough to be honest. I think this is important enough to have clear instructions somehow.

Reply:
@dianabarsan I made the instructions generic on purpose since I am not sure how much we want to make the docs Superset-specific. We leave the choice of a visualization tool to the end user and given we bundled Superset as part of CHT Sync before, we retained instructions about it. Superset is also in use within Medic but I am not sure about usage in the wider community. Should the detailed instructions be part of this [documentation](https://github.com/medic/cht-pipeline/issues/149) ticket or [this](https://github.com/medic/cht-sync/issues/142) one you created?

Reply:
I think one ticket is for discovery and implementation and the other one is for documentation. I think we need to figure out implementation before documentation.